docs/src/domains/steps/non-production-custom-domains.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Set up a custom domain on your non-production environments
+sidebarTitle: Non-production custom domains
+weight: 2
+description: Learn how to set up custom domains on your staging and development environments
+betaFlag: true
+---
+
+{{< premium-features/tiered "Elite and Enterprise" >}}
+
+You can add a custom domain to a staging or development environment without modifying your [routes configuration](../../define-routes/_index.md).
+
+To do so, you need to attach the new non-production custom domain
+to an existing production custom domain.
+
+{{< note title="Example" >}}
+
+You have two environments, a production environment and a staging environment.
+You've added the `example.com` custom domain to your production environment.
+
+You can only create the `staging.example.com` custom domain if you attach it to the `example.com` custom domain. 
+
+You can then access your staging environment through `staging.example.com`
+and still access your production environment through `example.com`.
+
+{{< /note >}}
+
+If you have multiple production environments,
+when you create a non-production custom domain,
+Platform.sh retrieves the `{default}` placeholder in your [routes configuration](../../define-routes/_index.md).
+It then replaces it with the correct production custom domain
+according to what you specify [in the setup command](#add-a-custom-domain-to-a-non-production-environment).
+
+Note that you must be a project admin to add non-production custom domains.
+You can add up to 5 non-production custom domains per project.
+
+If you delete a production custom domain,
+all of the attached non-production custom domains are deleted too.
+
+If you downgrade from an Elite or Enterprise plan to a Professional plan,
+all of the custom domains set on non-production environments are automatically removed.
+Downgrading your plan doesn't affect custom domains set on your production environments.
+
```

Comment:
Something I just learned today (not sure it's the right place to add this so feel free to move it):

> For Dedicated Gen 2 projects, this feature can only be used on the Grid environments, and not the dedicated ones. It is however possible to add a custom domain on a dedicated environment, but only via filing a support ticket.

Reply:
❓ I added the information, but how does it work for DG3 environments then? Do users need to go through support as well, or is the feature simply not available to them?